packages/functional-tests/pages/products/components/paymentInformation.ts

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Page } from '@playwright/test';
+
+export type CreditCard = {
+  name: string;
+  number: string;
+  expirationDate: string;
+  cvc: string;
+  zip: string;
+};
+
+export class PaymentInformationPage {
+  constructor(readonly page: Page) {}
+
+  get fullNameTextbox() {
+    return this.page.getByTestId('name');
+  }
+
+  get creditCardNumber() {
+    return this.page
+      .frame({ url: /elements-inner-card/ })
+      ?.getByPlaceholder('Card number');
```

Comment:
The optional chaining is unfortunate and needed due to the return type of 'frame'. 

In the case when it's nullish, the user will most likely get the following error, resulting from the 'Paynow' or 'Update' buttons not enabling.
<img width="700" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/3b9c34a8-d857-4e02-84fa-d79ef5885f3d">

my thoughts were that this is good enough because the test can't continue very far without the card information being filled in, but I'm open to further suggestion.